### DISCUSSION TITLE
CRUD Controller for enum value

### DISCUSSION THREAD

**Discussion by woutvdpol** [Laravel-Backpack/community-forum#678] [Q&A (Help)] [answered]
Hi, I have a table called contents, it consists of id, page (enum), title, description and index.
I have the whole contents CRUD controller right now. 
Is it possible to make a CRUD controller for the Contents table but only where the page is a specific enum value?
So on the 'ContentCRUDController' I get overview of every content, but on the 'HomeCRUDController' I only want the things from the Content table where page = 'HOME'.

I was wondering if this is possible and how?
Thanks in advance

**Comment by jcastroa87** [comment, marked as answer]
Hi @woutvdpol i'm not very sure if i correctly understand the requirement you raise, but if what you indicate is that you need to generate as many controllers as your model has enums, you could think about applying a global scope to restrict, for example, the results of the list that will be managed by that controller, here is an example of "addGlobalScope" in [our documentation](https://backpackforlaravel.com/docs/6.x/crud-how-to#how-to-filter-the-options-in-a-select-field-1)

Cheers

**Reply by woutvdpol** [reply]
Ah ok, I have tried it with `CRUD::addClause('where','page','TERMS');` but this gets everything out of the Contents table and then filters it on where page is terms. I am trying to get the addGlobalScope to work `CRUD::addGlobalScope('contents', function (Builder $builder) {
            $builder->where('page', 'TERMS');
        });`

Like this, but this shows everything and does not even filter on anything. Am I doing something wrong with the addGlobalScope?

**Reply by woutvdpol** [reply]
I think ill go ahead and use the addClause thing, it works almost as I want it to.

Thx for your response
